docs/Berger_POPC_test_case/comp_JMelcrProg/OP_buildH.out

```diff
@@ -0,0 +1,84 @@
+# OP_name            resname atom1 atom2  OP_mean OP_stddev OP_stem
```

Comment:
We could remove this file since it's removed in the `launch.sh`

Reply:
Done!